modules/boss/src/main/ts/ephox/boss/api/DomUniverse.ts

```diff
@@ -41,6 +41,13 @@ export default (): Universe<SugarElement, Document> => {
     Attribute.setAll(destination, as);
   };
 
+  const isSpecial = (element: SugarElement<Node>) => {
```

Comment:
`versions.txt` minor version bump for boss?

Reply:
Added